example_configs/wikijs.md

```diff
@@ -0,0 +1,64 @@
+# Configuration for WikiJS
+Replace `dc=example,dc=com` with your LLDAP configured domain.
+### LDAP URL
+```
+ldap://lldap:3890
+```
+### Admin Bind DN
+```
+cn=admin,ou=people,dc=example,dc=com
+```
+or 
+```
+cn=readonlyuser,ou=people,dc=example,dc=com
+```
+### Admin Bind Credentials
+```
+ADMINPASSWORD
+```
+or
+```
+READONLYUSERPASSWORD
+```
+### Search Base
+```
+ou=people,dc=example,dc=com
+```
+### Search Filter
+If you wish the permitted users to be restricted to just the `wiki` group: 
+```
+(&(memberof=cn=wiki,ou=groups,dc=example,dc=com)(|(uid={{username}})(mail={{username}))(objectClass=person))
+```
+If you wish any of the registered LLDAP users to be permitted to use WikiJS:
+```
+(&(|(uid={{username}})(mail={{username}))(objectClass=person))
+```
+### Use TLS
+Left toggled off
+### Verify TLS Certificate
+Left toggled off
+### TLS Certificate Path
+Left blank
+### Unique ID Field Mapping
+```
+uid
+```
+### Email Field Mapping
+```
+mail
+```
+### Display Name Field Mapping
+```
+givenname
```

Comment:
That's just the first name. The display name should be displayName

Reply:
I've tried `displayName` and it's probably a WikiJS thing but it gives `???` as the name.  Using givenname at least returns,  as you've pointed out, the users first name.
![Screenshot from 2023-01-21 11-56-39](https://user-images.githubusercontent.com/81871508/213865866-f677b638-b5bf-4beb-b29e-b0e3b9f1c416.png)

Reply:
Sorry to necro, but was this ever investigated? My users don't have the first name set, so I have to use displayName, but it ends up resolving to ???.

Reply:
You can try "cn" maybe